### PR TITLE
Adding assets required in cards app.

### PR DESCRIPTION
**Pull request by rahulpsingh**
## Description
Adding assets required in cards microapp.

## Testing
I have added these assets into test app and tested them.

## Checks
- [ ] Requires documentation update
- [x] Requires lib version update

## Versioning impact
- [ ] **MAJOR** incompatible API changes.
- [x] **MINOR** adds functionality in a backwards-compatible manner.
- [ ] **PATCH** backwards-compatible bug fixes.

### CODE REVIEW

File: src/icons/atm.svg [hunk 1]
```diff
@@ -0,0 +1,79 @@
+<svg viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (jasonadt) [inline src/icons/atm.svg]: Please add a `class="icon"` attribute to this `<svg>` element. 

Same comment for all other new SVG assets in this PR.

File: src/icons/lanaCardATM.svg [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<svg viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (jasonadt) [inline src/icons/lanaCardATM.svg]: this comment is not about this line but about the file name itself: please rename to: `lanaCardAtm.svg`

File: src/icons/searchATM.svg [hunk 3]
```diff
@@ -0,0 +1,33 @@
+<svg viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (jasonadt) [inline src/icons/searchATM.svg]: Similar comment for this file name, please rename to: `searchAtm.svg`

File: src/index.js [hunk 4]
```diff
@@ -9,6 +9,7 @@ import ArrowDownBoldIcon from './icons/arrowDownBold.svg';
 import ArrowDownLineIcon from './icons/arrowDownLine.svg';
 import ArrowUpBoldIcon from './icons/arrowUpBold.svg';
 import ArrowUpLineIcon from './icons/arrowUpLine.svg';
+import ATMIcon from './icons/atm.svg';
```
Comment (jasonadt) [inline src/index.js]: (nit, naming)  please rename this to: `AtmIcon`

File: src/index.js [hunk 5]
```diff
@@ -111,6 +113,12 @@ import InfoLineIcon from './icons/infoLine.svg';
 import InstagramIcon from './icons/instagram.svg';
 import InsuranceIcon from './icons/insurance.svg';
 import LanaCardIcon from './icons/lanaCard.svg';
+import LanaCardATMIcon from './icons/lanaCardATM.svg';
```
Comment (jasonadt) [inline src/index.js]: (nit, naming) please rename this to: `LanaCardAtmIcon`

File: src/index.js [hunk 6]
```diff
@@ -165,6 +174,7 @@ import ScanIcon from './icons/scan.svg';
 import ScanSuccessImage from './icons/scanSuccess.svg';
 import ScanFailureImage from './icons/scanFailure.svg';
 import SearchIcon from './icons/search.svg';
+import SearchATMIcon from './icons/searchATM.svg';
```
Comment (jasonadt) [inline src/index.js]: (nit, naming) please rename this to: `SearchAtmIcon`

**Review by jasonadt — CHANGES_REQUESTED** [review]

**Review by jantocv — APPROVED** [review]
❤️  thank you!!!!

**Review by TanMar23 — APPROVED** [review]

File: src/index.js [hunk 7]
```diff
@@ -38,15 +39,16 @@ import CarInsuranceIcon from './icons/carInsurance.svg';
 import CashIcon from './icons/cash.svg';
 import CashPaymentBillIcon from './icons/cashPaymentBill.svg';
 import CashPaymentIcon from './icons/cashPayment.svg';
-import Chat from './icons/chat.svg';
-import CheckboxChecked from './icons/checkboxChecked.svg';
-import CheckboxUnchecked from './icons/checkboxUnchecked.svg';
+import ChatIcon from './icons/chat.svg';
+import CheckboxCheckedIcon from './icons/checkboxChecked.svg';
```
Comment (jasonadt) [inline src/index.js:43]: Thanks for renaming these, we will need to be careful to remember to update the imports (and markup) used for this icon in the UI Library (same for the next line, and same for any other older icons that are being renamed in this PR)

**Review by jasonadt — APPROVED** [review]
👍  ship it! 🎉

File: src/index.js (hunk 7 above)
Comment (rahulpsingh) [reply src/index.js:43]: Yes, exactly!! We maybe should make it a major version!!

**Pull request merged**